etna/core/api.py

```diff
@@ -0,0 +1,70 @@
+from django.core.exceptions import PermissionDenied
+from django.http import Http404
+from requests import (
+    HTTPError,
+    JSONDecodeError,
+    RequestException,
+    Timeout,
+    TooManyRedirects,
+    get,
+)
+
+
+class APIClientError(Exception):
+    def __init__(self, message, response=None):
+        super().__init__(message)
+        self.response = response
+
+
+class JSONAPIClient:
```

Comment:
Is this the best place for this? It feels like the naming would suggest it is the Wagtail API.

Maybe move to something like `etna/core/lib.py` or `etna/core/json_api_client.py`?